src/os/os.js

```diff
@@ -5,15 +5,18 @@ goog.require('goog.events.EventTarget');
 goog.require('goog.userAgent');
 goog.require('goog.userAgent.product');
 goog.require('goog.userAgent.product.isVersion');
+goog.require('os.dispatch.dispatcher');
 goog.require('os.mixin.closure');
 goog.require('os.ol.license');
 
 
 /**
+ * TODO <AT>deprecated Please use goog.require('os.dispatch.dispatcher').getInstance() instead
+ *
  * The global event dispatcher.
  * @type {goog.events.EventTarget}
  */
-os.dispatcher = new goog.events.EventTarget();
+os.dispatcher = os.dispatch.dispatcher.getInstance();
```

Comment:
If you use `get/set` for this and make those calls pass through to `getInstance` and `setInstance`, then any existing code that does `os.dispatcher = <new value>` will correctly update the instance.

Reply:
Awesome idea -- but got blocked by how closure treats the "provide('blah')" namespace. So, as discussed, I put some tech debt into the code to use the new os.Dispatcher in the transforms but keep it pointed at this os.dispatcher for now for backward compatibility. Will re-update os.Dispatcher to use its own "instance" variable again later.